packages/theme_tailor_annotation/lib/src/theme_tailor.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:meta/meta_meta.dart';
+
+import '../theme_tailor_annotation.dart';
+
+const tailor = Tailor();
+
+/// {@template theme_tailor.theme_tailor}
+/// ### Tailor
+///
+/// {@endtemplate}
+///
+@Target({TargetKind.classType})
+class Tailor {
+  const Tailor([this.props = defaultProps, this.themes = defaultThemes]);
+
+  static const String light = 'light';
+  static const String dark = 'dark';
+
+  static const List<String> defaultThemes = [light, dark];
+  static const List<TailorProp> defaultProps = [
+    TailorProp('lucky', [7, 8]),
```

Comment:
Do we need this weird defaults here?

Reply:
No, and it is reworked here https://github.com/Iteo/theme_tailor/pull/3 so I think it can stay as it is in this PR

Reply:
Ok